app/src/main/java/com/example/netflixclone/MainActivity.kt

```diff
@@ -49,14 +53,19 @@ class MainActivity : ComponentActivity() {
 }
 
 @Composable
-@Preview
+@Preview(device = Devices.PIXEL_4_XL)
 fun Greeting() {
-    Box(
+    Column(
         modifier = Modifier
             .fillMaxSize()
-            .background(Color.Black),
+            .background(color = Color.Black)
     ) {
-        Column() {
+        Column(
```

Comment:
Gave this child a weight, so it will take up the rest of the screen after bottom bar takes up as much height as it needs.